app/views/rails_db/shared/_assets.html.erb

```diff
@@ -0,0 +1,111 @@
+<!-- CSS Assets -->
+<%= insert_css_file 'rails_db/foundation.min.css' %>
+<%= insert_css_file 'rails_db/foundation_and_overrides.css' %>
+<%= insert_css_file 'codemirror/codemirror.css' %>
+<%= insert_css_file 'codemirror/codemirror/themes/pastel-on-dark.css' %>
+<%= insert_css_file 'codemirror/codemirror/addons/hint/show-hint.css' %>
+<%= insert_css_file 'rails_db/app.css' %>
+<%= insert_css_file 'rails_db/sidebar.css' %>
+<%= insert_css_file 'rails_db/pagination.css' %>
+<%= insert_css_file 'rails_db/table.css' %>
+<%= insert_css_file 'rails_db/forms.css' %>
+<%= insert_css_file 'rails_db/search.css' %>
+
+<!-- JavaScript Assets -->
+<%= insert_js_file 'codemirror/codemirror.js' %>
+<%= insert_js_file 'codemirror/codemirror/modes/sql.js' %>
+<%= insert_js_file 'codemirror/codemirror/addons/hint/show-hint.js' %>
+<%= insert_js_file 'codemirror/codemirror/addons/hint/sql-hint.js' %>
+<%= insert_js_file 'rails_db/jquery.js' %>
+<%= insert_js_file 'rails_db/jquery.cookie.js' %>
+<%= insert_js_file 'rails_db/rails-ujs.js' %>
+<%= insert_js_file 'rails_db/foundation.min.js' %>
+<%= insert_js_file 'rails_db/sticky.js' %>
+<%= insert_js_file 'rails_db/show_hide_columns.js' %>
+<%= insert_js_file 'rails_db/search.js' %>
+
+<script>
+$(function(){
```

Comment:
as I understand this is part of application.js

maybe what we can do is to place this into a separate file and embed here and load there. 

so we will not copy paste code 

wdyt?